src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabServerConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import jakarta.inject.Inject;
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+public class GitLabServerConfig extends PageAreaImpl {
+
+    private Control serverName = control(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/name"));
+    private Control serverUrl = control(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl"));
```

Comment:
use the pagearea to obtain the relative path?

```suggestion
    private Control serverName = control("servers", "name"));
    private Control serverUrl = control("servers", "serverUrl"));
```